core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3634,7 +3642,8 @@ public Integer getValue() {
           () -> master.getNumberOfPaths());
 
       final String ufsDataFolder = Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS);
-      final UnderFileSystem ufs = ufsManager.getRoot().getUfs();
+      // Note: Ufs client resource is never closed
+      final UnderFileSystem ufs = ufsManager.getRoot().acquireUfsClientResource().get();
```

Comment:
Does it make sense to be able to put the root ufs into maint? If we do so, we will get errors for the metrics, but that should be acceptable since it is down. You could do the `get` inside each of the gauge definitions and close them after getting the info. However if the interval is too short, it might be difficult for the user to understand.

Reply:
I'll move it inside